src/util.cc

```diff
@@ -53,6 +58,12 @@
 
 static std::atomic_int seq = {0};  // Sequence number for diagnostic filenames.
 
+#ifndef DISABLE_SINGLE_EXECUTABLE_APPLICATION
+static bool single_executable_application_code_loaded = false;
```

Comment:
It's probably a good idea to put them into the `per_process` namespace, which is generally what we do for per-process global variables. Or if more succinct names are preferred maybe `node::per_process::sea::...`? (I think SEA is probably going to be well-understood in the code base sooner or later anyway, just like ESM).